vmdb/app/models/user.rb

```diff
@@ -176,7 +176,14 @@ def role_allows?(options={})
     return false if self.miq_user_role.nil?
     feature = MiqProductFeature.find_by_identifier(options[:identifier])
     identifiers = {:identifiers => [options[:identifier]]}
-    feature.try(:hidden) ? miq_user_role.allows_any?(identifiers) : miq_user_role.allows?(options)
+    if feature.try(:hidden)
+      parent = MiqProductFeature.feature_parent(options[:identifier])
+      parent_feature = MiqProductFeature.find_by_identifier(parent)
```

Comment:
These two methods should just be in MiqProductFeature directly.  Maybe a `find_parent_indentifier` or something.